war/src/main/less/modules/table.less

```
@@ -139,8 +142,11 @@
     }
   }
 
-  &__button, &__icon {
-    svg, .build-status-icon__wrapper, img {
+  &__button,
+ &__icon {
+    svg,
+ .build-status-icon__wrapper,
+ img {
```

This looks like a mess in the new version. Compare with:

```suggestion
  &__button,
  &__icon {
    svg,
    .build-status-icon__wrapper,
    img {
```